### PR TITLE
fix: test-python and deploy-with-kustomize job

### PR DESCRIPTION
**Pull request by bnormington**
- **fix: passing store-coverage-artifacts from job to command**
- **fix: adding cluster-name pass through to deploy-with-kustomize**
- **fix: added resource_class to test-python**

### CODE REVIEW

File: src/jobs/deploy-with-kustomize.yml [hunk 1]
```diff
@@ -23,19 +23,38 @@ parameters:
...
-      cluster-name: goodway-<< parameters.environment >>
-      image-name: $AWS_ECR_REGISTRY_URL/microservice/<< parameters.app-name >>
-      kustomize-path: << parameters.kube-path >>/overlays/<< parameters.environment >>
-      namespace: << parameters.app-name >>-<< parameters.environment >>
-      wait-for-rollout: << parameters.wait-for-rollout >>
-      post-deploy-steps: << parameters.post-deploy-steps >>
-      deployment-resource-name: << parameters.deployment-resource-name >>
-      notification-channel: << parameters.notification-channel >>
+
+  - unless:
```
Comment (jeffcap1) [inline src/jobs/deploy-with-kustomize.yml:36]: Just for my knowledge, what is the benefit of using `unless` vs `when` for this step?

**Review by jeffcap1 — APPROVED** [review]
👀 👍

File: src/jobs/deploy-with-kustomize.yml (hunk 1 above)
Comment (bnormington) [reply src/jobs/deploy-with-kustomize.yml:36]: They are boolean opposite of each other and there is no `else` so the unless works as the else case.

**Pull request merged**